src/main/resources/pdiresources/jobs/migration.kjb

```diff
@@ -286,10 +341,190 @@
   <pass_batchid>N</pass_batchid>
   <shared_objects_file/>
   <entries>
+    <entry>
+      <name>Start</name>
+      <description/>
+      <type>SPECIAL</type>
```

Comment:
how about moving the creation and dropping of the old_encounter_type table steps into its transformation and just loading the transformation here since i guess we will end up running more migration transformations here

Reply:
I've tried that and it didn't work. 
Creating needs to happen in the beginning and drop in the end, these steps seem to start at the same time leading to unpredictable results. 
The only way to coordinate this is to move the steps to a job, which always runs sequentially.